SlickEngagement_Plugin.php

```diff
@@ -307,22 +310,22 @@ public function removeSemicolons($value)
 
     // Fetches the Page Boot Data from the server
     //TODO: if we find that `/page-boot-data` requests are reduced enough to always use origin, we can add headers to avoid hitting CloudFlare
-    private function fetchBootData($siteCode) 
+    private function fetchBootData( $siteCode )
     {
         $protocol = ((!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] != 'off') || $_SERVER['SERVER_PORT'] == 443) ? "https://" : "http://";
         $page_url = $protocol . $_SERVER['HTTP_HOST'] . $_SERVER['REQUEST_URI'];
```

Comment:
This data won't be available in the cron task, so I'm not sure how this is working?

Reply:
Ah good point. I'll fix this